index.html

```diff
@@ -52,6 +52,127 @@
                 </div>
             </div>
         </div>
+        <section class="grid-container-card">
+          
+            <div class="grid-card">
+                <div class="responsive-backgroundImage">
+                    <img src="/Image/portfolio.jpg" alt="portfolio"/>
+                </div>
+                <div class="identification">
+                    <h3>Tonic</h3>
+                    <div class="flex-container">
+                        <div class="title-bolder">CANOPY</div>
+                        <div class="point"></div>
+                        <div class="title-non-bolder">Back End Dev</div>
+                        <div class="point"></div>
+                        <div class="title-non-bolder">2015</div>
+                    </div>
+                    <div class="text">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+                      <ul class="grid-container">
+                        <li><a class="skills" href="#">html</a></li>
+                        <li><a class="skills" href="#">css</a></li>
+                        <li><a class="skills" href="#">javascript</a></li>
+                      </ul>  
+                    <button type="button" class="btn">See Project</button>
+                </div>
+              </div>
+              <div class="grid-card">
+                <div class="responsive-backgroundImage">
+                    <img src="/Image/second.png" alt="portfolio"/>
+                </div>
+                <div class="identification">
+                    <h3>Multi-Post<br>Stories</h3>
+                    <div class="flex-container">
+                        <div class="title-bolder">CANOPY</div>
+                        <div class="point"></div>
+                        <div class="title-non-bolder">Back End Dev</div>
+                        <div class="point"></div>
+                        <div class="title-non-bolder">2015</div>
+                    </div>
+                    <div class="text">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+                      <ul class="grid-container">
+                        <li><a class="skills" href="#">html</a></li>
+                        <li><a class="skills" href="#">css</a></li>
+                        <li><a class="skills" href="#">javascript</a></li>
+                      </ul>  
+                    <button type="button" class="btn">See Project</button>
+                </div>
+              </div>
+              <div class="grid-card">
+                <div class="responsive-backgroundImage">
+                    <img src="/Image/SnapshootCard3.png" alt="portfolio"/>
```

Comment:
All your images are not showing kindly put a full stop in front of your link to fix
hint: `src="./Image/second.png"`

![image](https://user-images.githubusercontent.com/43564324/159125320-4fe0480e-762a-4b33-94c8-0f1080cefa23.png)